integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/RandomNameCreator.java

```diff
@@ -14,17 +14,19 @@
 @Component
 public class RandomNameCreator {
 
+    public static final String PREFIX = "autotest-";
+
     @Inject
     private RandomNameGenerator generator;
 
     @Value("${" + CLOUD_PROVIDER + ":MOCK}")
     private CloudPlatform cloudPlatform;
 
     public String getRandomNameForResource() {
-        return cloudPlatform.name().toLowerCase() + '-' + generator.next().replaceAll("_", "-");
+        return PREFIX + cloudPlatform.name().toLowerCase() + '-' + generator.next().replaceAll("_", "-");
```

Comment:
Please add this prefix for all of Purgable.deletable implementations

Reply:
Wow, that same method was implemented like 15 times? I better fix that.